packages/math-input/src/components/keypad/mobile-keypad.tsx

```diff
@@ -0,0 +1,158 @@
+import {StyleType} from "@khanacademy/wonder-blocks-core";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+import ReactDOM from "react-dom";
+
+import Key from "../../data/keys";
+import {View} from "../../fake-react-native-web/index";
+import {Cursor, KeypadConfiguration, KeyHandler} from "../../types";
+
+import Keypad from "./index";
+
+/**
+ * This is the v2 equivalent of v1's ProvidedKeypad. It follows the same
```

Comment:
To allow us to hot-swap the v1 and v2 keypads, this wrapper needs to have the same API as `/keypad-legacy/provided-keypad.tsx`